projects/fal/pyproject.toml

```diff
@@ -41,6 +41,8 @@ dependencies = [
     "pydantic!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.*,!=2.4.*,<3",
     # serve=True dependencies
     "fastapi>=0.99.1,<1",
+    # For compatability with fastapi <1
+    "starlette-exporter==0.17.1",
```

Comment:
Hm, what do you mean by <1 ? fastapi 1.0 is not out yet, they are in 0.1** right now.

Reply:
oops sorry, the conflict is with the transitive dependency on starlette. In our internal fal tooling we pin to fastapi==0.98, which requires a starlette version that is incompatible with the latest starlette-exporter.
We can let the installer handle dependency resolution and omit the requirement for starlette-exporter.

Reply:
it turns out that this is only a problem for fastapi==0.98. Since we are higher than that, it's ok and we can remove this requirement.